SimFastTiming/FastTimingCommon/src/BTLElectronicsSim.cc

```diff
@@ -77,6 +81,26 @@ void BTLElectronicsSim::run(const mtd::MTDSimHitDataAccumulator& input,
       float finalToA2 = finalToA1 + times[1];
       finalToA1 += times[0];
 
+      // --- Estimate the time uncertainty due to photons from earlier OOT hits in the current BTL cell
+      if (smearTimeForOOTtails_) {
+        float rate_oot = 0.;
+        // Loop on earlier OOT hits
+        for (unsigned int ibx = 0; ibx < 9; ++ibx) {
+          if ((it->second).hit_info[2 * iside][ibx] > 0.) {
+            float npe_oot = CLHEP::RandPoissonQ::shoot(hre, (it->second).hit_info[2 * iside][ibx]);
+            rate_oot += npe_oot * exp((it->second).hit_info[1 + 2 * iside][ibx] / ScintillatorDecayTime_) /
```

Comment:
can slightly speed up the math here by storing `ScintillatorDecayTimeInv_ = 1./ScintillatorDecayTime_` once and then multiplying by `ScintillatorDecayTimeInv_` (rather than dividing, which is slower)

Reply:
Done